k8s-daemonset/k8s/hello-world.yml

```diff
@@ -50,7 +50,7 @@ metadata:
 spec:
   selector:
     app: hello
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
Alex pointed out that we shouldn't be exposing the hello service regardless (in the other example I think we only access it via linkerd) - he suggested removing it